ocaml/idl/datamodel.ml

```diff
@@ -1216,6 +1216,8 @@ let _ =
     ~doc:"The update precheck stage failed: the server is of an incorrect version." ();
   error Api_errors.update_precheck_failed_out_of_space ["update"; "available_space"; "required_space "]
     ~doc:"The update precheck stage failed: the server does not have enough space." ();
+  error Api_errors.update_precheck_failed_gpgkey_not_imported ["update"]
+    ~doc:"The update precheck stage failed: RPM package validation requires a GPG key that is not imported on the host." ();
```

Comment:
Just a nit pick for native speakers to review: is a _key that is not imported on this host_ the right tense? I would have expected _key has not been imported into this host_ or _key is not present on this host_.